lib/service/fetch_basket_item.rb

```diff
@@ -0,0 +1,10 @@
+module Shop
+  class FetchBasketItem
+    def call(basket_item)
+      return unless basket_item
+      BASKET.find do |basket_item|
+        basket_item == basket_item.to_i
```

Comment:
Hmm this will not works, you compare twice the same value basket_item with itself